workflows/reference/calc-gene-lengths.wdl

```diff
@@ -39,9 +39,12 @@ workflow calc_gene_lengths {
         Int? max_retries
     }
 
-    call util.calc_gene_lengths as calc { input: gtf=gtf, max_retries=max_retries }
+    call util.calc_gene_lengths as calc_gene_lens { input:
```

Comment:
Do we need to abbreviate this?

Reply:
Yes, I didn't realize why at first. The encompassing workflow name is `calc_gene_lengths`. A little surprised they're considered the same namespace. Not the biggest fan of this abbreviation, and I'm realizing a better solution might be to expand "calc"->"calculate" in the workflow name. Or in the task name? Either one would get rid of the name collision.

Reply:
That said, renaming the task is likely easier.